pkg/sql/colexec/execgen/testdata/type_template

```diff
@@ -0,0 +1,76 @@
+template
+package main
+
+
+func main() {
+  a_int(3, 4)
+  a_float(3.3, 4)
+  a_bool(true, 4)
+}
+
+// execgen:template<T>
+// execgen:instantiate<int>
+// execgen:instantiate<float>
+// execgen:instantiate<bool>
+func a (a T, b int, T interface{}) {
+  fmt.Println(a)
+  // execgen:call<T>
```

Comment:
I see there's one down here. Can we only use this for template variable pass through?

Reply:
Oh, this was just an example/typo/thing that I forgot to remove. It doesn't do anything.